package.json

```diff
@@ -28,16 +28,18 @@
   "scripts": {
     "audit:all": "npm audit --if-present",
     "audit:fix:all": "npm audit --fix --if-present",
+    "build": "docker compose build",
     "ci:all": "npm ci --if-present",
-    "dev": "docker-compose up --build",
-    "dev:detach": "npm run dev -- --detach",
     "housekeeping": "npm run update:all && npm run install:all && npm run lint:fix:all && npm run audit:fix:all && npm run spellcheck",
     "postinstall": "husky install",
     "install:all": "npm install --if-present",
     "lint:all": "npm run lint --workspaces --if-present",
     "lint:fix:all": "npm run lint --fix --workspaces --if-present",
     "load": "node utils/mock-data-loader/re-insert-mocks.js",
     "spellcheck": "cspell lint --gitignore --no-must-find-files --unique --no-progress --show-suggestions --color '**/*'",
+    "start": "docker compose up",
```

Comment:
Why are we splitting these out of interest?

Reply:
Have unifed.